.github/workflows/main.yml

```diff
@@ -92,11 +92,11 @@ jobs:
           projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
           onlyChanged: "!startsWith(github.ref, 'refs/heads/main')" # Use TurboSnap for PR builds
           externals: |
-            - './packages/nimble-components/.storybook/public/**'
+            - './packages/storybook/.storybook/public/**'
             - './packages/nimble-tokens/dist/icons/svg/**'
             - './packages/nimble-tokens/source/styledictionary/properties/**'
           storybookBuildDir: ./packages/site/dist/storybook
-          storybookBaseDir: ./packages/nimble-components
+          storybookBaseDir: ./packages/storybook
           exitOnceUploaded: true # Do not wait for test results
           exitZeroOnChanges: true # Option to prevent the workflow from failing
 
```

Comment:
@m-akinc I've seen it before but in the future please don't resolve the "useful" comment style comments until the PR is ready to merge. Unlike AzDo you can't see them without manually expanding them which is tedious